modules/ctconfiguration/ctconfiguration.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * Class CTConfiguration
+ *
+ * @property $bootstrap
+ */
+class CTConfiguration extends Module
+{
+    /**
+     * CTConfiguration constructor.
+     */
+    public function __construct()
+    {
+        $this->name    = 'ctconfiguration';
+        $this->tab     = 'front_office_features';
+        $this->version = '1.0.0';
+        $this->author  = 'Community Theme';
+
+        parent::__construct();
+
+        $this->displayName = $this->l('Community Configuration');
+        $this->description = $this->l('Configuration for community theme blocks and content.');
+        $this->ps_versions_compliancy = array('min' => '1.6', 'max' => '1.6.1.4');
```

Comment:
As patch versions are a little bit faster these days and code is pretty simple here I'd suggest to keep min as 1.6 and max as _PS_VERSION_